static/src/stylesheets/module/facia-garnett/_container--video.scss

```diff
@@ -301,6 +301,19 @@ $video-width-desktop: 700px;
     }
 }
 
+.video-playlist__item--paid-for.video-playlist__item .fc-item__kicker,
+.video-playlist__item--paid-for.video-playlist__item .video-overlay__duration {
+    color: $labs-main !important;
```

Comment:
Linked to my last comment about `!important` (https://github.com/guardian/frontend/pull/21685#discussion_r306852193), is it necessary on the new rules in this file?

Reply:
For this ones we need them, but it won't be a problem of conflicting with other `!important`s because we are doing it on the newly created wrapper class `.video-playlist__item--paid-for`. Open to other suggestions

Reply:
I'd still be concerned that it has the potential to cause conflicts for developers in the future who may need to alter these colour properties `.video-playlist__item--paid-for`. I assume the rules are overridden by a more specific rule elsewhere?

Reply:
I have now added more specific css classes to each item